packages/loader/driver-utils/src/error.ts

```diff
@@ -3,11 +3,13 @@
  * Licensed under the MIT License.
  */
 
+// eslint-disable-next-line import/no-deprecated
```

Comment:
If DriverErrorType is being deprecated, does it need to be imported?

Reply:
See my reply to your other comment. We still have to use the deprecated type in a number of places to avoid making this change breaking for consumers.